multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/OrphanedFilesCleaner.java

```diff
@@ -35,69 +37,77 @@ public class OrphanedFilesCleaner {
+    // it is started with
     @Scheduled(fixedRate = 30, timeUnit = TimeUnit.MINUTES)
     public void run() {
         if (configuration.getApplicationInstanceIndex() != SELECTED_INSTANCE_FOR_CLEANUP) {
             return;
         }
 
         var timestamp = LocalDateTime.now();
-        var oneHourAgo = timestamp.minusHours(1);
+        var oneAndHalfHoursAgo = timestamp.minusHours(1)
+                                          .minusMinutes(30);
+        var halfHourAgo = timestamp.minusMinutes(30);
         try {
-            LOGGER.debug(MessageFormat.format(Messages.GETTING_FILES_CREATED_AFTER_0, oneHourAgo));
-            var files = fileService.listFilesCreatedAfter(oneHourAgo);
+            LOGGER.info(MessageFormat.format(Messages.GETTING_FILES_CREATED_AFTER_0_AND_BEFORE_1, oneAndHalfHoursAgo, halfHourAgo));
+            var files = fileService.listFilesCreatedAfterAndBefore(oneAndHalfHoursAgo, halfHourAgo);
             var fileIdsToFiles = files.stream()
-                                      .collect(Collectors.toMap(FileEntry::getId, entry -> entry));
-
-            var historicVariables = getHistoricAppArchiveIDs(oneHourAgo);
+                                      .collect(Collectors.toMap(FileEntry::getId, Function.identity()));
 
-            filterFilesWithStartedOperations(fileIdsToFiles, historicVariables);
+            var archiveIdsWithAssociatedOperation = getHistoricAppArchiveIDs(oneAndHalfHoursAgo, halfHourAgo);
+            filterFilesWithStartedOperations(fileIdsToFiles, archiveIdsWithAssociatedOperation);
 
             if (fileIdsToFiles.isEmpty()) {
                 LOGGER.info(Messages.NO_ORPHANED_FILES_TO_DELETE);
                 return;
             }
-            LOGGER.debug(MessageFormat.format(Messages.DELETING_ORPHANED_FILES_0, fileIdsToFiles.size(), fileIdsToFiles.keySet()));
-            for (var orphanedFile : fileIdsToFiles.values()) {
-                fileService.deleteFile(orphanedFile.getSpace(), orphanedFile.getId());
-            }
-        } catch (FileStorageException e) {
-            throw new SLException(e, Messages.COULD_NOT_DELETE_ORPHANED_FILES_MODIFIED_AFTER_0, oneHourAgo);
+            LOGGER.info(MessageFormat.format(Messages.DELETING_ORPHANED_FILES_0, fileIdsToFiles.size(), fileIdsToFiles.keySet()));
+            fileIdsToFiles.forEach((fileId, file) -> deleteFileSafely(file));
+        } catch (Exception e) {
+            throw new SLException(e,
+                                  Messages.COULD_NOT_DELETE_ORPHANED_FILES_MODIFIED_AFTER_0_AND_BEFORE_1,
+                                  oneAndHalfHoursAgo,
+                                  halfHourAgo);
         }
     }
 
-    private List<String> getHistoricAppArchiveIDs(LocalDateTime startedAfter) {
+    private List<String> getHistoricAppArchiveIDs(LocalDateTime startedAfter, LocalDateTime startedBefore) {
         LOGGER.debug(MessageFormat.format(Messages.GETTING_OPERATIONS_STARTED_AFTER_0, startedAfter));
```

Comment:
Maybe add the startedBefore value to the logger here as well

Reply:
done